### PR TITLE
Adding missing property & other minor changes

### PR DESCRIPTION
**Pull request by ChrisMBarr**
- the `permissionLevel` property was missing
- `needsPermission` and `isSupported` are properties, not functions
- Spelling fixes in comments

### CODE REVIEW

**Comment by vvakame** [comment]
*notifyjs/notifyjs.d.ts*

to author(@soundTricker). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

File: notifyjs/notifyjs.d.ts [hunk 1]
```diff
@@ -10,20 +10,25 @@ declare var Notify: {
      * Check is permission is needed for the user to receive notifications.
      * @return true : needs permission, false : does not need
      */
```
Comment (soundTricker) [inline notifyjs/notifyjs.d.ts]: Please modify jsdoc comment to fit changing.

File: notifyjs/notifyjs.d.ts [hunk 2]
```diff
@@ -10,20 +10,25 @@ declare var Notify: {
...
+     * @param onPermissionDeniedCallback  A callback for permission is denied.
      */
     requestPermission(onPermissionGrantedCallback?: ()=> any, onPermissionDeniedCallback? : ()=> any) : void;
 
     /**
      * return true if the browser supports HTML5 Notification
-     * @param true : the browser supports HTML5 Notification, false ; the browswer does not supports HTML5 Notification.
+     * @param true : the browser supports HTML5 Notification, false ; the browser does not supports HTML5 Notification.
      */
-    isSupported() : boolean;
```
Comment (soundTricker) [inline notifyjs/notifyjs.d.ts]: Same as the comment above.

**Comment by soundTricker** [comment]
@chrismbarr 
I've commented to file.
Please fix some issues, see also comment.

* Modify test file ( notifyjs/notifyjs-tests.ts ) to pass the test.
* Modify header comment.
  * Modify notify.js's version to reference your version.
* Modify JSDocs to fit changing.

File: notifyjs/notifyjs.d.ts (hunk 1 above)
Comment (chrismbarr) [inline notifyjs/notifyjs.d.ts]: I'm not sure what you mean by this... Can you explain?

**Comment by chrismbarr** [comment]
https://github.com/chrismbarr/DefinitelyTyped/commit/75f03d73ce8a8a10f4594bd6500d6f207b2e8191

**Comment by chrismbarr** [comment]
@vvakame @soundTricker Check this now, I think it might be good to merge.  If not, let me know.

**Comment by chrismbarr** [comment]
Bump...  Is this good now?

**Comment by vvakame** [comment]
@chrismbarr wow, sorry. it merged! thanks mate :+1:

**Pull request merged**